config/locales/devise.en.yml

```
@@ -12,7 +12,7 @@ en:
       invalid: "Invalid %{authentication_keys} or password."
       locked: "Your account is locked."
       last_attempt: "You have one more attempt before your account is locked."
-      not_found_in_database: "Invalid %{authentication_keys} or password."
+      not_found_in_database: "Invalifd %{authentication_keys} or password."
```

Aqui acho que entrou um f sem querer o/ 
```suggestion
      not_found_in_database: "Invalid %{authentication_keys} or password."
```

---

Obrigado @JorgeLAB, ajuste realizado.